src/codecs/cats/whisker.c

```diff
@@ -0,0 +1,125 @@
+#include <string.h>
+#include <assert.h>
+
+#include "whisker.h"
+
+#ifdef RS41
+#define HARDWARE_ID 0x7d00
```

Comment:
Are these some pre-allocated IDs for whisker data? This could deserve a comment and maybe even a link to CATS documentation where these are defined!